sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -28,11 +30,23 @@ public class CommunicationTestEnvironment : TestEnvironment
 
         public string CommunicationTestPhoneNumber => GetVariable(AzurePhoneNumber);
 
+        public string SkipSmsTest => GetVariable(SkipSmsTestEnvironmentVariableName);
+
+        public string SkipPhoneNumbersTest => GetVariable(SkipPhoneNumbersTestEnvironmentVariableName);
+
         public virtual string ExpectedTestPackagesEnabled { get { return TestPackagesEnabledDefaultValue; } }
 
         public bool ShouldIgnoreTests => TestPackagesEnabled != TestPackagesEnabledDefaultValue
             && TestPackagesEnabled != ExpectedTestPackagesEnabled;
 
+        public bool ShouldIgnoreSMSTests => TestPackagesEnabled != TestPackagesEnabledDefaultValue
+            && TestPackagesEnabled != ExpectedTestPackagesEnabled
+            || bool.Parse(SkipSmsTest);
+
+        public bool ShouldIgnorePhoneNumbersTests => TestPackagesEnabled != TestPackagesEnabledDefaultValue
```

Comment:
I am wondering if you need to check the `TestPackagesEnabled` here, isn't checking `SkipPhoneNumbersTest` enough?
* if  `TestPackagesEnabled` is not equal to phonenumbers, everything will be skipped anyway
* and if phonenumbers is enabled, then you just need to skip selected tests based on `SkipPhoneNumbersTest` value

Reply:
I still need to check 'TestPackagesEnabled != TestPackagesEnabledDefaultValue' because one one can still trying to run just SMS test in Prod manually by going to the pipeline and set the TestPackagesEnabled flag. Without this check , that functionality will be broken.